.github/workflows/continous-integration.yml

```diff
@@ -273,6 +284,15 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN }}
 
+    - name: Notify Sentry about the release
+      env:
+        GITHUB_TAG: ${{ github.ref }}
+        SENTRY_ORG: rasahq
+        SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      run: |
+        VERSION=${GITHUB_TAG/refs\/tags\//}
+        ./scripts/inform_sentry_about_release.sh
```

Comment:
optional: What's the benefits of using the script? I'd prefer to keep the CI workflows self-contained unless it de-duplicates things.

Reply:
you're right, will move the commands here